pom.xml

```diff
@@ -182,6 +182,7 @@
         <scopt.version>4.1.0</scopt.version>
         <slf4j.version>1.7.36</slf4j.version>
         <snakeyaml.version>1.33</snakeyaml.version>
+        <jackson.datatype.version>2.12.3</jackson.datatype.version>
```

Comment:
can we move this close to  <jackson.version> ? and why does these version different ?